src/components/common/Avatar/index.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <img :src="src" :class="classes"/>
```

Comment:
We need accessible image tag with alt attribute. 
user name may be passed to this component as props and used in alt attribute value.

Reply:
Yes, I am working on that now.... Will add it soon and remove WIP status